### ISSUE TITLE
1.20 Beta: Editing contents of results file breaks navigation.

### ISSUE THREAD

**Issue by phyllisstein** [atom/find-and-replace#953]
### Prerequisites

* [x] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: http://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode
    * Followed all applicable steps in the debugging guide: http://flight-manual.atom.io/hacking-atom/sections/debugging/
    * Checked the FAQs on the message board for common solutions: https://discuss.atom.io/c/faq
    * Checked that your issue isn't already filed: https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom
    * Checked that there is not already an Atom package that provides the described functionality: https://atom.io/packages

### Description
In the current beta version of Atom, navigating to a find-in-project result and changing the contents of the file causes the package to throw an error (`TypeError: Cannot read property '0' of undefined`) on subsequent attempts to jump to search results. Here's a quick screencast, with the v1.20 beta in safe mode, demonstrating what I mean

![screenflow](https://user-images.githubusercontent.com/2207980/29953804-1997b97c-8ea1-11e7-8766-a980829e4693.gif)

### Steps to Reproduce

1. Perform a project-wide search with the find-and-replace panel.
2. Click a hit in the "Project Find Results" view.
3. Edit and save the file.
3. Click another hit in the "Project Find Results" view.
4. No bueno.

**Expected behavior:** Atom should jump to the second hit without throwing an error.

**Actual behavior:** Error pops up and we're stuck where we are.

**Reproduces how often:** 100%

### Versions
```
~ ❯❯❯ atom-beta --version
Atom    : 1.20.0-beta5
Electron: 1.6.9
Chrome  : 56.0.2924.87
Node    : 7.4.0

~ ❯❯❯ apm-beta --version
apm  1.18.4
npm  3.10.10
node 6.9.5 x64
python 2.7.10
git 2.14.1

~ ❯❯❯ sw_vers
ProductName:	Mac OS X
ProductVersion:	10.12.6
BuildVersion:	16G29
```

### Additional Information
💋

**Comment by phyllisstein** [comment]
(N.B. also that this is reproducible with a build from Atom's latest `master` as well.)

**Comment by iolsen** [comment]
@maxbrunsfeld can you look at this? Seems likely to be related to your recent work.

**Comment by scholtzm** [comment]
Is the fix from #954 included in stable v1.20.0? It's still happening on my end.

**Comment by Ben3eeE** [comment]
@scholtzm No, the fix from #954 was not included in Atom 1.20.0. See https://github.com/atom/find-and-replace/pull/954#issuecomment-327426074

**Comment by scholtzm** [comment]
Oops, missed that comment. Thanks!
